### DISCUSSION TITLE
3-finger horizontal swipe to navigate columns

### DISCUSSION THREAD

**Discussion by KiaraGrouwstra** [YaLTeR/niri#224] [Ideas]
niri now allows using the touchpad with three fingers swiping vertically to switch between workspaces.
if i recall correctly paperwm had a similar thing with horizontal swiping to navigate between window columns within a workspace.
it would be cool if niri supported a similar operation (/ exposed gestures such as to support that).
i do think paperwm had it scroll kind of more smoothly than a binary switch, which could be harder but seems maybe cool as well.

**Comment by YaLTeR** [comment]
Yes https://github.com/YaLTeR/niri/issues/100
